Content.Server/Nutrition/EntitySystems/SmokingSystem.Cigar.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Server.Nutrition.Components;
+using Content.Shared.Chemistry.Components.SolutionManager;
+using Content.Shared.Interaction;
+using Content.Shared.Smoking;
+using Content.Shared.Temperature;
+using Robust.Shared.GameObjects;
+
+namespace Content.Server.Nutrition.EntitySystems
+{
+    public partial class SmokingSystem
```

Comment:
ngl this being a partial class instead of just another system is really ugly

it being in 'nutrition' is also sus but thats fine

Reply:
> ngl this being a partial class instead of just another system is really ugly

Personally, I really prefer making use of partial classes rather than creating new, almost-empty, useless systems.
The idea is that in the future, there will be different smoking methods (vapes, smoking pipes) which will also be partial files for this class.

To me, this is not ugly. Quite the opposite, to me this feels clean and well-organized.
All relevant logic to smoking now resided in the same class, while still maintaining good organization.

> it being in 'nutrition' is also sus but thats fine

That's how it was before, I could change it if you really want tho.

Reply:
I'm fine with it IMO.

Reply:
mmm after more consideration i still dont really like it but its fine